packages/stems/src/components/Popup/Popup.tsx

```diff
@@ -162,12 +163,13 @@ const getComputedOrigins = (
 const getAdjustedPosition = (
   top: number,
   left: number,
-  wrapperRect: DOMRect
+  wrapperRect: DOMRect,
+  containerRect: Pick<DOMRect, 'width' | 'height'>
 ): { adjustedTop: number; adjustedLeft: number } => {
   if (!wrapperRect) return { adjustedTop: 0, adjustedLeft: 0 }
 
-  const containerWidth = window.innerWidth - CONTAINER_INSET_PADDING
-  const containerHeight = window.innerHeight - CONTAINER_INSET_PADDING
+  const containerWidth = containerRect.width - CONTAINER_INSET_PADDING
```

Comment:
this is the one change that leaks a bit out to existing impls - we were originally always computing the adjusted position based on window, not the container, and only using the container to get the computed origins

I *think* this is more correct with the intended behavior? @raymondjacobson I think you added the container stuff, does this seem right? How would I test the usecase for the containerRef outside of my current one to make sure I didn't break anything?